tmva/sofie/inc/TMVA/RModel_BasicBinaryOp.hxx

```diff
@@ -62,18 +92,19 @@ public:
 
    std::string Generate(std::string OpName){
       OpName = "op_" + OpName;
+
       if (fShape.empty()) {
-         throw std::runtime_error("TMVA SOFIE Add called to Generate without being initialized first");
+         throw std::runtime_error("TMVA SOFIE Binary Op called to Generate without being initialized first");
       }
       std::stringstream out;
       // int length = 1;
       // for(auto& i: fShape){
       //    length *= i;
       // }
       size_t length = ConvertShapeToLength(fShape);
-      out << "\n//------ Add\n";
+      out << "\n//------ " + s->Name()+"\n";
```

Comment:
Also here we should use std::string. Maybe we define the functions to return a std::string instead of doing 
`std::string(BinaryOperatorTrait<T,Op>::Op())`